examples/cgroup-rstat-flushing.yaml

```diff
@@ -0,0 +1,31 @@
+metrics:
+  counters:
+    - name: cgroup_rstat_flush_total
+      help: Total number of times cgroup rstat were flushed (recorded per level)
+      labels:
+        - name: level
+          size: 4
+          decoders:
+            - name: uint
+    - name: cgroup_rstat_locked_total
+      help: Total number of times rstat lock was obtainted and contended state
+      labels:
+        - name: contended
+          size: 4
+          decoders: # contended boolean converted to 0 and 1
+            - name: uint
+    - name: cgroup_rstat_locked_yield
+      help: Number of times rstat lock was obtainted again after yield and contended state
+      labels:
+        - name: contended
+          size: 4
+          decoders: # contended boolean converted to 0 and 1
+            - name: uint
+    - name: cgroup_rstat_lock_contended
```

Comment:
`cgroup_rstat_lock_contentions_total`, I think.

Reply:
This map was removed after improvement to encode this as labels.